Tests/ModelTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Part of the Joomla GSoC Webservices Project
+ *
+ * @copyright  Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE
+ */
+
+namespace Joomla\Entity\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Joomla\Entity\Tests\Models\User;
+
+require __DIR__ . '/../vendor/autoload.php';
```

Comment:
This isn't required - you're going to have this file loaded automatically by the phpunit bootstrap xml file